content/en/docs/Reference/kairos-factory.md

```diff
@@ -17,75 +17,353 @@ For the newcomer or someone who simply needs an immutable OS with k3s and edgeVP
+2025-02-27T13:33:49Z INF Preparing ISO image root tree...
+2025-02-27T13:33:49Z INF Copying /tmp/geniso2408076146 source to /tmp/auroraboot-iso220411365/iso
+2025-02-27T13:33:49Z INF Starting rsync...
+2025-02-27T13:33:49Z INF Finished syncing
+2025-02-27T13:33:49Z INF Finished copying /tmp/geniso2408076146 into /tmp/auroraboot-iso220411365/iso
+2025-02-27T13:33:49Z INF Creating EFI image...
+2025-02-27T13:33:49Z INF Detected Flavor: ubuntu
+2025-02-27T13:33:49Z INF Ubuntu based ISO detected, copying grub.cfg to /EFI/ubuntu/grub.cfg
+2025-02-27T13:33:49Z INF Creating file system image /tmp/auroraboot-iso220411365/iso/boot/uefi.img with size 4Mb
+2025-02-27T13:33:49Z INF Creating squashfs...
+2025-02-27T13:33:53Z INF Creating ISO image...
+$ ls build      
+config.yaml  kairos.iso  kairos.iso.sha256  netboot  temp-rootfs
+```
+
+
+## Configuration
+
+kairos-init can generate both core and standard images, and standard images can be bundled with either k3s or k0s and any version of the software that you want.
+
+It can also prepare OCI artifacts for [Trusted Boot](../Architecture/trustedboot.md) which are slimmer than the usual ones, as they have size limitations plus we dont want to ship things like grub or dracut in them as they are useless.
+
+
+
+Here is a list of flags, explanation and what are the possible and default values
+
+| Flag         | Explanation                                              | Possible values             | Default value   |
+|--------------|----------------------------------------------------------|-----------------------------|-----------------|
+| --version    | Set version of the artifact that we are building         | Any                         | None (REQUIRED) |
+| -l           | Sets the log level                                       | info,warn,debug,trace       | info            |
+| -s           | Sets the stage to run                                    | install, init, all          | all             |
+| -m           | Sets the model                                           | generic, rpi3, rpi4         | generic         |
+| -v           | Sets the variant                                         | core, standard              | core            |
+| -k           | Sets the Kubernetes provider                             | k3s,k0s                     | k3s             |
+| -r           | Sets registry and org (Checks this repo when upgrading)  | Any valid registry and org  | quay.io/kairos  |
+| -t           | Sets Trusted Boot on                                     | true,false                  | false           |
+| -f           | Sets the framework version to use                        | Any valid framework version | latest          |
+| --fips       | Use FIPS framework for FIPS 140-2 compliance images      | true,false                  | false           |
+| --k8sversion | Sets the provider version to use                         | Any valid provider version  | latest          |
```

Comment:
can't we already say: k3s or k0s ?

Reply:
well, those are providers no? So any valid provider version is correct and it opens to have more providers (rke2 maybe?) so I think this is correct and concise.